### ISSUE TITLE
Fix flaky tests

### ISSUE THREAD

**Issue by zerolab** [wagtail/wagtail-vector-index#8]
`test_get_split_content_splits_longer_content` is a bit flaky and sometimes `splits == 1` resulting in 

```
    def test_get_split_content_splits_longer_content():
        content_length = 340
        split_length = 200
        body = fake.text(max_nb_chars=content_length)[:content_length]
        instance = ExamplePageFactory.create(title="a", body=body)
        splits = instance._get_split_content(split_length=split_length, split_overlap=0)
>       assert len(splits) == 2
E       AssertionError: assert 1 == 2
E        +  where 1 = len(['a\nPainting right various sort. Western medical cut third of song hotel social. Early young with from form dog hand.\nStyle though rest home shoulder agency. Own use look blue film loss enough.'])
```

most prononunce in CI, but had it locally a few times

**Comment by zerolab** [comment]
I believe this was fixed by #24
/cc @tm-kn

**Comment by tm-kn** [comment]
Haven't seen evidence of tests failing randomly in a while. Closing.
